addons/website/static/src/interactions/menu/mega_menu_dropdown.js

```diff
@@ -0,0 +1,103 @@
+import { Interaction } from "@website/core/interaction";
+import { registry } from "@web/core/registry";
+
+class MegaMenuDropdown extends Interaction {
+
+    static selector = "header#top";
+    static dynamicContent = {
+        ".o_mega_menu_toggle": {
+            "t-on-mouseenter": "onHoverMegaMenu",
+            "t-on-mousedown": "onTriggerMegaMenu",
+            "t-on-keyup": "onTriggerMegaMenu",
+        },
+        "_root": {
+            "t-on-mousedown": "onTriggerExtraMenu",
+            "t-on-keyup": "onTriggerExtraMenu",
+        },
```

Comment:
What about a comment that mentions that those are delegated on that specific class ?